I_have_no_vinaigrette.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+import random
+import re
+
+
+def days_between(d1, d2):
+    d1 = datetime.datetime.strptime(d1, "%Y-%m-%d")
+    d2 = datetime.datetime.strptime(d2, "%Y-%m-%d")
+    return abs(d1 - d2)
+
+
+def i_have_no_vinaigrette():
+    date1, date2 = input("Enter two dates in format YYYY-MM-DD and date1 smaller than date2: ").split()
```

Comment:
When you write code, you should aim to write it as "pure" a possible - avoid IO (input output) operations near logic. why is that? if you want to use this logic elsewhere - for example, receive a path from a place other than the standart input,